docs/data-sources/compute_flavors.md

```diff
@@ -44,6 +51,6 @@ resource "huaweicloud_compute_instance" "instance" {
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - Specifies a data source ID in UUID format.
+* `id` - Specifies a data source ID.
```

Comment:
No `Specifies` for `Attributes Reference` parameters description.

Reply:
Indicates?